_pages/blog/tags.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Tags
+layout: page
+section: tags
+---
+
+{% capture counts_with_tags_string %}{% for tag in site.tags %}{{ tag[1] | size | prepend:"000000" | slice:-6,6 }}:{{ tag[0] }}{% unless forloop.last %},{% endunless %}{% endfor %}{% endcapture %}
```

Comment:
really there isn't a better way to format a number?? 😮

Reply:
Nope. Apparently not 🤷

Reply:
We probably can, but that would need to write a template helper function.

Reply:
Yes, of course with a plugin we could make this cleaner.